server/README.md

```diff
@@ -0,0 +1,88 @@
+# Quiz App Server
+
+This directory contains the backend code for the Quiz App, built using Python FastAPI. The backend handles the API endpoints, including health checks, and serves as the core engine for the application's logic.
+
+## Project Structure
+
+```
+server/
+│
+├── Pipfile
+├── Pipfile.lock
+├── .env
+├── main.py
+└── README.md
+```
+
+### Files & Directories
+
+- **`Pipfile`**: Manages the Python dependencies for the project using `pipenv`.
+- **`Pipfile.lock`**: Locks the specific versions of the dependencies to ensure consistency across different environments.
+- **`.env`**: Stores environment variables, including the port number for the FastAPI server.
+- **`main.py`**: The entry point for the FastAPI application, where the API routes and health check endpoint are defined.
+- **`README.md`**: This document, providing an overview of the backend setup and instructions for development.
+
+## Requirements
+
+- **Python**: 3.12
+- **pipenv**: For managing Python dependencies.
+- **direnv**: For managing environment variables with the `.env` file.
+
+## Setup Instructions
+
+### 1. Clone the Repository
+
+```bash
+git clone https://github.com/yourusername/quiz-app.git
+cd quiz-app/backend
+```
+
+### 2. Install Dependencies
+
+Ensure you have Python 3.12 installed. Then, install the required dependencies using `pipenv`.
+
+```bash
+pipenv install
+```
+
+### 3. Setup Environment Variables
+
+The project uses `direnv` to manage environment variables from the `.env` file. Ensure you have `direnv` installed and then run:
+
+```bash
+direnv allow
+```
+
+Make sure the `.env` file contains the following variable:
+
+```plaintext
+PORT=8000
+```
+
+You can change the port number as needed.
+
+### 4. Run the Application
+
+Start the FastAPI server with the following command:
+
+```bash
+pipenv run uvicorn main:app --reload
```

Comment:
Since this is fastapi, don't you think installing fastapi and using `fastapi dev main.py` is better?